### PR TITLE
Create ru.json

### PR DESCRIPTION
**Pull request by partizan007**
Russian localisation

### CODE REVIEW

File: locales/ru.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "Published with": "Опубликовано",
+  "Subscribe": "Подписаться",
+  "Subscribe to": "Подписаться на",
+  "Subscribed!": "Вы подписанны!",
+  "Subscribe to our newsletter": "Подпишитесь на обновления сайта!",
+  "You've successfully subscribed to": "Вы успешно подписались!",
+  "with the email address": "email адрес",
+  "Email cannot be blank.": "Email не может быть пустым!",
+  "Get the latest posts delivered right to your inbox.": "Получайте последни обновления прямо на почту!",
+  "Stay up to date! Get all the latest & greatest posts delivered straight to your inbox.": "Stay up to date! Get all the latest & greatest posts delivered straight to your inbox.",
```
Comment (eddiesigner) [inline locales/ru.json]: Can you please add this translation? :)

File: locales/ru.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "Recommended for you": "Рекомендуем",
+  "Featured": "ТОП",
+  "Page not found": "Страница не найдена",
+  "Unfortunately the page you were looking for could not be found.": "Unfortunately the page you were looking for could not be found.",
```
Comment (eddiesigner) [inline locales/ru.json]: Can you please add this translation too? :)

File: locales/ru.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  "Back to home": "На главную",
+  "Go to the home page": "На главную страницу",
+  "Recent articles": "Последние посты",
+  "No recent articles found :(": "Последние статьи не найдены :(",
+  "1 min read": "1 минута на чтение", 
+  "% min read": "% минут на чтение",
+  "No posts": "No posts",
```
Comment (eddiesigner) [inline locales/ru.json]: And this one ;)

File: locales/ru.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  "No posts found": "Ничего не найдено",
+  "Apparently there are no posts at the moment, check again later.": "Apparently there are no posts at the moment, check again later.",
```
Comment (eddiesigner) [inline locales/ru.json]: And finally this one ;)

File: locales/ru.json (hunk 1 above)
Comment (partizan007) [reply locales/ru.json]: Ok

**Comment by stereohorse** [comment]
@partizan007, will you continue translation? If not, I can pick it up.

**Comment by eddiesigner** [comment]
I will close this PR since there is a russian localization already.

**Pull request closed without merging**